src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/BaseTest.cs

```diff
@@ -407,7 +407,8 @@ protected bool StreamCompare (Stream stream1, Stream stream2)
 		protected string GetPathToLatestBuildTools (string exe)
 		{
 			var path = Path.Combine (AndroidSdkPath, "build-tools");
-			foreach (var dir in Directory.GetDirectories (path, "*", SearchOption.TopDirectoryOnly).OrderByDescending (x => new Version (Path.GetFileName (x)))) {
+			foreach (var dir in Directory.GetDirectories (path, "*", SearchOption.TopDirectoryOnly).OrderByDescending (x => Path.GetFileName (x))) {
```

Comment:
Pity this will fails when sorting `{ "2", "10" }`, because it sorts "stringly", not numerically.

Fortunately this is unlikely to be a problem *now*, because we're likely to always have two digit numbers.  (But when we start hitting Build-tools 100, watch out!)

Unfortunately, this also sorts `{ "29.0.0-rc2", "29.0.0-rc1", "29.0.0" }` in that order, such that the `-rc2` will be preferred *first*, over the "stable" 29.0.0 version.